mmv1/third_party/terraform/tests/resource_gkeonprem_vmware_node_pool_test.go.erb

```diff
@@ -0,0 +1,188 @@
+<% autogen_exception -%>
+package google
+<% unless version == 'ga' -%>
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccGkeonpremVmwareNodePool_vmwareNodePoolUpdate(t *testing.T) {
+  // VCR fails to handle batched project services
```

Comment:
I think we can remove this comment + the SkipIfVcr below!

Reply:
Removed